consensus/src/creation/mod.rs

```diff
@@ -55,16 +55,23 @@ async fn create_unit<H: Hasher>(
     round: Round,
     creator: &mut Creator<H>,
     create_lag: &DelaySchedule,
-    mut can_create: bool,
+    mut try_create: bool,
```

Comment:
Both `can_create` and `try_create` are actually slightly confusing. This bool captures whether we need to wait for the delay to pass or not. A better name would be `delay_passed` or something similar.

Reply:
Ok, after thinking a little more... This is wrong because we use a single bool for two different things. What should be done:
1) Call this bool `skip_delay` and remove `mut`
2) Create a new bool in the function body `delay_passed`.
3) Adjust the rest of the code inside the function.

Reply:
How about passing explicitly `initial_delay` instead of that flag: `create_lag` if normal, 0 if catching up, and move create as last step then?

Reply:
I don't think this will do what we want. The problem here is that there should be really two functions:
1) create_unit_no_earlier_than_delay
2) create_unit_asap

And we are trying to capture both with one function.

Reply:
current version: we have two separate functions now, `keep_processing_units_until` and `create_unit`. The former, retrieves new units from provided channel until some given timeout. The latter, is a loop `try create unit -> if failed download one more unit`. This version is more verbose. What do you think?